core/src/main/scala/kafka/coordinator/transaction/TransactionStateManager.scala

```diff
@@ -600,8 +589,6 @@ class TransactionStateManager(brokerId: Int,
                 recordsPerPartition,
                 updateCacheCallback,
                 delayedProduceLock = Some(newMetadata))
-
-              trace(s"Appended new metadata $newMetadata for transaction id $transactionalId with coordinator epoch $coordinatorEpoch to the local transaction log")
```

Comment:
Hm. It is useful to have the `transactionalId` in that log line. In my PR, I rephrased it to `Attempting to append new metadata..`. That is more accurate, and also preserves the transactionalId. So we know that the log append was attempted, and can take the investigation from there..